app/controller/settings.js

```diff
@@ -0,0 +1,33 @@
+var fs = require('fs');
+var express = require('express');
+
+var router = express.Router();
+
+var configFile = '../test.json';
+var configData = require(configFile);
+
```

Comment:
what the hell are these two includes for? just use const {config} = require('../config.json').